extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -502,13 +492,65 @@ private function getCuratedContentQualityForWiki( $wikiID ) {
 	 *
 	 * @return bool
 	 */
-	static function onCuratedContentSave( $sections ) {
-		self::purgeMethod( 'getList' );
```

Comment:
`self::purgeMethod` is inherited from `self`(`CuratedContentController`) → `WikiaController` → `WikiaDispatchableObject`. It's used in few places. I am purging this URL in one `SquidUpdate` → line 528.
```
// Purge all sections list getter URL - no additional params
[ self::getUrl( 'getList' ) ]
```